CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+ComplianceAsCode Contribution Guidelines
+===
+
+We aim to create a welcoming environment for all contributors regardless of their age, gender, experience, nationality, or employer. However, to foster smooth collaboration we are asking contributors to adhere to the following rules.
+
+Issues and Pull Requests
+---
+
+Pull requests can’t be merged by the author, a review from another person is needed. If anybody raises material objections to the pull request, those objections should be addressed before the PR merge.
+
+If a pull request has received feedback from reviewers, and the original author hasn’t touched it for more than 14 days, the PR may be closed (it is still possible to re-open it if the author catches up later).
+
```

Comment:
```suggestion
- Ideally pull request feedback would be addressed within 14 days. This helps us move the project forward and helps prevent the need for major rebasing of pull requests. Everyone goes on vacation or has reasons this won't always be possible. If that's the case, please let us know in the pull request to avoid it being closed for no response. Even if your pull request is closed, worst case you'll need to re-open it upon your return (no code will be lost!).

```

Reply:
The original formulation was indeed too strong. I like your suggestions, maybe consider adding a sentence that if a PR is inactive for no apparent reason for a prolonged period of time, we place a "14 days until we close it" warning as a comment.

Reply:
I would make this 30 days rather than 14 days as it gives more time for vacations and such.

Reply:
OK, 30 days it is.